float32[] config
int32 targetObs
---
bool response
bool collisionFree
int32 collisionObject
